src/NeoSharp.Core/Messaging/MessageHandlerProxy.cs

```diff
@@ -1,45 +1,138 @@
-﻿using System;
+﻿using NeoSharp.Core.Caching;
+using NeoSharp.Core.DI;
+using NeoSharp.Core.Logging;
```

Comment:
It is a good practices to put System namespaces at first.

Reply:
I use alphabetical order

Reply:
vert minor thing here, we should do System first, then other namespaces in alphabetical

Reply:
The result of remove and sort is this. i use this option every time.

Reply:
Gotcha, you don't have 
![image](https://user-images.githubusercontent.com/454196/41175823-603e6624-6b2c-11e8-9c2a-c177dfc61874.png)